R/infrastructure-git.R

```diff
@@ -59,6 +59,8 @@ use_git <- function(message = "Initial commit", pkg = ".") {
 #' @param auth_token Provide a personal access token (PAT) from
 #'   \url{https://github.com/settings/tokens}. Defaults to the \code{GITHUB_PAT}
 #'   environment variable.
+#' @param host GitHub API host to use. Override with your GitHub enterprise
+#'   hostname, for example, "github.hostname.com/api/v3".
```

Comment:
Why do you give the full path here? I think it should just be the protocol and host name